airbyte-integrations/connectors/source-flexport/metadata.yaml

```diff
@@ -1,23 +1,28 @@
 data:
+  allowedHosts:
+    hosts:
+      - TODO # Please change to the hostname of the source.
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: false
   connectorSubtype: api
   connectorType: source
   definitionId: f95337f1-2ad1-4baf-922f-2ca9152de630
-  dockerImageTag: 0.1.0
+  dockerImageTag: 0.4.0
```

Comment:
Why are you bumping 3 minor versions and not only one?

Reply:
https://semver.org/

Reply:
> [semver.org](https://semver.org/)

Thank you!